e3sm_diags/plot/annual_cycle_zonal_mean_plot.py

```diff
@@ -147,38 +120,14 @@ def _add_colormap(
     _configure_titles(ax, title)
 
     # Configure x and y axis.
-    #    # --------------------------------------------------------------------------
-    #    _configure_x_and_y_axes(ax, X_TICKS, None, None, parameter.current_set)
-    #    ax.set_xlim(X_LIM)
-    #
-    #    if parameter.plot_plevs:
-    #        plev_ticks = parameter.plevs
-    #        plt.yticks(plev_ticks, plev_ticks)
-    #
-    #    # For default plevs, specify the pressure axis and show the 50 mb tick
-    #    # at the top.
-    #    if (
-    #        not parameter.plot_log_plevs
-    #        and not parameter.plot_plevs
-    #        and parameter.plevs == DEFAULT_PLEVS
-    #    ):
-    #        plev_ticks = parameter.plevs
-    #        new_ticks = [plev_ticks[0]] + plev_ticks[1::2]
-    #        new_ticks = [int(x) for x in new_ticks]
-    #        plt.yticks(new_ticks, new_ticks)
-    #
-    #    plt.ylabel("pressure (mb)")
-    #    ax.invert_yaxis()
+    # --------------------------------------------------------------------------
+    plt.xticks(time, X_TICKS)
+    lat_formatter = LatitudeFormatter()  # type: ignore
+    ax.yaxis.set_major_formatter(lat_formatter)
+    ax.tick_params(labelsize=8.0, direction="out", width=1)
+    ax.xaxis.set_ticks_position("bottom")
+    ax.yaxis.set_ticks_position("left")
```

Comment:
I added this block of code from the old plotter because it was missing here.

Reply:
I think it fixes the "refine axis config for plot" todo item in this [comment](https://github.com/E3SM-Project/e3sm_diags/pull/798#issuecomment-2016055403).